front/src/components/post/PostDetailPage.vue

```diff
@@ -57,23 +60,63 @@ export default {
           depth2: "",
           depth3: "",
         },
+        likes: [],
         createdAt: "",
         comments: [],
       },
     };
   },
+  computed: {
+    postDate() {
+      return this.$moment(this.post.createdAt).fromNow();
+    },
+    liked() {
+      return this.post.likes.some(
+        (like) =>
+          like.memberId === this.$store.getters["member/getMembers"] &&
+          like.postId === this.post.id,
+      );
+    },
```

Comment:
```javascript
    checkLike(like) {
      return (
        like.memberId === this.$store.getters["member/getMembers"] &&
        like.postId === this.post.id
      );
    },
```
methods에 이 함수를 추가한 뒤에,
```suggestion
    liked() {
      return this.post.likes.some(this.checkLike);
    },
```
이렇게 바꿔보는 건 어때요?
deletePostLike()에도 적용할 수 있을거예요.

메소드 명이 뭔가 딱 오는 좋은 느낌은 아니네요. 😅